Add new troubleshooting section for generating abi for maps (#149)
AntelopeIO/cdt: docs/08_troubleshooting/index.md

The failure is stating that `eosio.system` `init` action was not called yet. The `init` action is implemented by the `void init(uint64_t, symbol)` function. The first parameter is the version, this should always be `0` for now, until a new version of `init` will be created that handles more information.
The second parameter is the system's symbol (i.e. for main net this is `EOS`). If you followed the [BIOS Boot Sequence](https://docs.eosnetwork.com/docs/latest/tutorials/bios-boot-sequence) tutorial and created a system with the default symbol `SYS` then `SYS` shall be used as the system's symbol in the `init` action. It is whatever symbol you as the chain creator want to use in your `Antelope` based blockchain.

## Backward incompatible change in generating abi for maps

> In CDT version 3.0.1, there has been a backward incompatible change in generating ABI for std::map. The change is as follows:

Reviewer: This language might be clearer?
```suggestion
In CDT version 3.0.1, there has been breaking ABI change in std::map. The change is as follows:
```
